lib/models/navigation/page_data/email_auth_page_data.dart

```diff
@@ -0,0 +1,26 @@
+library email_auth_page_data;
```

Comment:
Each page we want to navigate to has a corresponding page data object that can be added to the app state's pagesData member, which in turn is used to create the list of pages that is passed in to the navigator

Reply:
ahhh thank you